applications/luci-app-mwan3/luasrc/model/cbi/mwan/ruleconfig.lua

```diff
@@ -15,62 +14,72 @@ local ipsets = util.split(util.trim(util.exec("ipset -n -L 2>/dev/null | grep -v
 m = Map("mwan3", translatef("MWAN Rule Configuration - %s", arg[1]))
 m.redirect = dsp.build_url("admin", "network", "mwan", "rule")
 
-mwan_rule = m:section(NamedSection, arg[1], "rule", "")
-mwan_rule.addremove = false
-mwan_rule.dynamic = false
+s = m:section(NamedSection, arg[1], "rule", "")
+s.addremove = false
+s.dynamic = false
 
-src_ip = mwan_rule:option(Value, "src_ip", translate("Source address"),
+o = s:option(ListValue, "family", translate("Internet Protocol"))
+o.default = "any"
+o:value("any", translate("IPv4 and IPv6"))
```

Comment:
I prefer use empty as "IPv4 and IPv6"
we treat “” as `IPv4 and IPv6`
and treat "any" as `IPv4 and IPv6`

Reply:
This is very easy to understand. If a rule does not specify a family, we think it is a dual stack
and In actual programming, this is very practical and effective